web/packages/teleterm/src/ui/AppUpdater/AutoUpdatesManagement.tsx

```diff
@@ -0,0 +1,321 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ReactNode, useState } from 'react';
+
+import { Alert } from 'design/Alert';
+import { CheckboxInput } from 'design/Checkbox';
+import { Stack } from 'design/Flex';
+import { Cog } from 'design/Icon';
+import Link from 'design/Link';
+import { RadioGroup } from 'design/RadioGroup';
+import Text from 'design/Text';
+import { UnreachableCluster } from 'gen-proto-ts/teleport/lib/teleterm/auto_update/v1/auto_update_service_pb';
+
+import {
+  AppUpdateEvent,
+  AutoUpdatesDisabled,
+  AutoUpdatesEnabled,
+  AutoUpdatesStatus,
+  Cluster,
+} from 'teleterm/services/appUpdater';
+import { RootClusterUri } from 'teleterm/ui/uri';
+
+import {
+  ClusterGetter,
+  clusterNameGetter,
+  findUnreachableClusters,
+} from './common';
+
+export function AutoUpdatesManagement(props: {
+  status: AutoUpdatesStatus;
+  updateEventKind: AppUpdateEvent['kind'];
+  clusterGetter: ClusterGetter;
+  changeManagingCluster(clusterUri: RootClusterUri | undefined): void;
+  onCheckForUpdates(): void;
+}) {
+  const { status } = props;
+
+  const unreachableClusters = findUnreachableClusters(status);
+  const getClusterName = clusterNameGetter(props.clusterGetter);
+  const content =
+    status.enabled === true
+      ? makeContentForEnabledAutoUpdates(status, getClusterName)
+      : makeContentForDisabledAutoUpdates(status);
+
+  return (
+    <>
+      {unreachableClusters.length > 0 && (
+        <Alert
+          width="100%"
+          mb={0}
+          kind="warning"
+          primaryAction={{
+            content: 'Refresh',
+            onClick: props.onCheckForUpdates,
+          }}
+          details="Unable to retrieve accepted client versions from some clusters. Compatibility with these clusters will not be verified."
+        >
+          Unreachable clusters
```

Comment:
It's fine that we skip which clusters failed in the widget view, however I feel like the details view should include info about which clusters we failed to connect to.